engine-standalone-storage/src/relayer_db/mod.rs

```diff
@@ -81,6 +82,7 @@ where
         block_timestamp: env::Timestamp::new(0),
         attached_deposit: 0,
         random_seed: H256::zero(),
+        prepaid_gas: NearGas::new(0),
```

Comment:
I think the default value for this should be 300_000_000_000_000. It is the maximum allowed prepaid gas and the most common value for transactions in practice as far as I know. You could probably even define a constant for this value so that you can simply reference it in all the places we use it

```rust
// env.rs

pub const DEFAULT_PREPAID_GAS: NearGas = NearGas::new(300_000_000_000_000);
```